convert-hf-to-gguf.py

```diff
@@ -228,8 +229,10 @@ def _get_part_names(self):
             return ("pytorch_model.bin",)
         return (f"pytorch_model-{n:05}-of-{self.num_parts:05}.bin" for n in range(1, self.num_parts + 1))
 
+        raise NotImplementedError(f'Architecture "{arch}" not supported!')
+
```

Comment:
This should have been removed along with the function it was part of in <https://github.com/ggerganov/llama.cpp/pull/6920/commits/02fd977fe138d7bf57b0cf53cae3491eb59e28ec>. As it is, it's unreachable.